### PR TITLE
dotCMS/core#24541  [UI] Order the experiment listing

### LINKED ISSUE #24541: [UI] Order the experiment listing
### Parent Issue

#24343

### User Story

- Order of Experiment Listing should be 1. Running, 2. Scheduled, 3, Draft, 4. Ended, 5. Archived

### Acceptance Criteria

- [ ] Order of Experiment Listing should be 1. Running, 2. Scheduled, 3, Draft, 4. Ended, 5. Archived

### Proposed Objective

User Experience

### Proposed Priority

Priority 2 - Important

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.

_No response_

### Assumptions & Initiation Needs

_No response_

### Quality Assurance Notes & Workarounds

_No response_

### PR DESCRIPTION
**Pull request by hmoreras**
### Proposed Changes
* Order of Experiment Listing should be 1. Running, 2. Scheduled, 3, Draft, 4. Ended, 5. Archived

### Screenshots

image

### CODE REVIEW

File: core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-list/components/dot-experiments-list-table/dot-experiments-list-table.component.html [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <div class="dot-experiment-list-table">
-    <ng-container *ngIf="groupedExperimentsCount; else noExperimentsTpl">
-        <ng-container *ngFor="let group of experimentGroupedByStatus | keyvalue">
-            <p-table [value]="group.value" styleClass="experiment-list">
+    <ng-container *ngIf="experimentsCount; else noExperimentsTpl">
+        <ng-container *ngFor="let group of experimentGroupedByStatus">
+            <p-table [value]="group.experiments" styleClass="experiment-list">
                 <ng-template pTemplate="caption">
-                    {{ group.key }}
+                    {{ group.status | lowercase | dm | uppercase }}
```
Comment (rjvelazco) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-list/components/dot-experiments-list-table/dot-experiments-list-table.component.html:6]: Why do we need the `lowercase` pipe before the `dm` pipe?

File: core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-list/store/dot-experiments-list-store.ts [hunk 2]
```diff
@@ -81,17 +81,33 @@ export class DotExperimentsListStore
...
+                    grouped.push({ status: DotExperimentStatusList[key], experiments: [] })
+                );
+
+                experiments
+                    .filter((experiment) => filterStatus.includes(experiment.status))
+                    .forEach((experiment) => {
+                        const status = grouped.find((item) => item.status === experiment.status);
+
+                        if (status) {
+                            status.experiments.push(experiment);
```
Comment (rjvelazco) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-list/store/dot-experiments-list-store.ts:99]: What do we use this `status` constant for? I think we are not using or returning it.

**Review by rjvelazco — CHANGES_REQUESTED** [review]

**Review by zJaaal — APPROVED** [review]

File: core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-list/store/dot-experiments-list-store.ts (hunk 2 above)
Comment (hmoreras) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-list/store/dot-experiments-list-store.ts:99]: I will change the name to `statusGroup` basically if is present we just need to push the experiment.

File: core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-list/components/dot-experiments-list-table/dot-experiments-list-table.component.html (hunk 1 above)
Comment (hmoreras) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-list/components/dot-experiments-list-table/dot-experiments-list-table.component.html:6]: just to make sure it match the properties in `Language.properties`

**Review by rjvelazco — APPROVED** [review]

**Review by oidacra — APPROVED** [review]

**Pull request merged**